### DISCUSSION TITLE
Connect in cascade or in series 2 SDM630

### DISCUSSION THREAD

**Discussion by jfery63** [arendst/Tasmota#15378] [Q&A] [answered]
Hello everyone,
I have 2 SDM630 and I would like to link them together in order to be able to read them both by tasmota and recover the data in MQTT in order to insert them in grafana.
I installed tasmota.bin and sensors.bin, but nothing to do, I only see one SDM630
Can you tell me what I need to do or how to connect the 2 SDM630s so that tasmota sees them both.
I thank you in advance for your help.
John
Ps: sorry for my English, but I go through a translation page.
Here is what I get for the moment, I would have to be able to see 2 sensors

![mqtt6](https://user-images.githubusercontent.com/103791027/163652840-18a51265-84ac-4754-9d07-ac93e5c5e2d0.jpg)

**Comment by barbudor** [comment, marked as answer]
The native SDM630 driver (USE_SDM630) supports only 1 SDM630 driver.

You may have more luck using the generic [Smart Meter](https://tasmota.github.io/docs/Smart-Meter-Interface/) from the [Scripting-Language](https://tasmota.github.io/docs/Scripting-Language/)

This [discussion](https://github.com/arendst/Tasmota/discussions/15320) about having multiple MODBUS meters may help you. See the answers from **gemu2015**

**Reply by jfery63** [reply]
hello, 
Thanks for the info, I'll try to see if I got anywhere with the link you gave me.
Good for you.
John
